clients/mobile/src/components/donation/PostCard.tsx

```diff
@@ -80,59 +96,61 @@ export const PostCard: React.FC<PostCardProps> = ({ post, updateHandler, detailH
   }), [dropdownPosition])
 
   return (
-    <>
       <View style={styles.card}>
-        {showHeader && <View style={styles.cardHeader}>
-          <View>
-            <Text style={styles.userName}>{userProfile.name}</Text>
-            <Text style={styles.postTime}>Posted on {formatDateTime(post.createdAt)}</Text>
-          </View>
-          <View style={styles.menuContainer}>
-            <View ref={iconRef} collapsable={false}>
-              <TouchableOpacity
-                onPress={handleToggleDropdown}
-                style={styles.iconContainer}
-              >
-                <Ionicons name="ellipsis-vertical" size={20} color={theme.colors.grey} />
-              </TouchableOpacity>
+        {showHeader &&
+          <View style={styles.cardHeader}>
+            <View>
+              <Text style={styles.userName}>{post.patientName}</Text>
+              <Text style={styles.postTime}>Posted on {formatDateTime(post.createdAt)}</Text>
             </View>
-            <Modal
-              visible={showDropdown}
-              transparent={true}
-              animationType="none"
-              onRequestClose={handleCloseDropdown}
-            >
-              <TouchableWithoutFeedback onPress={handleCloseDropdown}>
-                <View style={styles.modalOverlay}>
-                  <TouchableWithoutFeedback>
-                    <View style={getDropdownStyle()}>
-                      <TouchableOpacity
-                        onPress={handleUpdate}
-                        style={styles.dropdownItem}
-                      >
-                        <Text style={styles.dropdownText}>Update</Text>
-                      </TouchableOpacity>
-                      <TouchableOpacity
-                        onPress={handleCloseDropdown}
-                        style={styles.dropdownItem}
-                      >
-                        <Text style={styles.dropdownText}>Delete</Text>
-                      </TouchableOpacity>
-                    </View>
-                  </TouchableWithoutFeedback>
-                </View>
-              </TouchableWithoutFeedback>
-            </Modal>
+            {showOptions &&
+            <View style={styles.menuContainer}>
+              <View ref={iconRef} collapsable={false}>
+                <TouchableOpacity
+                  onPress={handleToggleDropdown}
+                  style={styles.iconContainer}
+                >
+                  <Ionicons name="ellipsis-vertical" size={20} color={theme.colors.grey} />
+                </TouchableOpacity>
+              </View>
+
+              <Modal
+                visible={showDropdown}
+                transparent={true}
```

Comment:
could we not just use transparent

Reply:
Updated [commit](https://github.com/CraftsmenLtd/BloodConnect/pull/295/commits/14e368f3cf7b2d686f72f9c468cbe55f9a276229)